packages/react-ui-ag/src/Listings/MobileSRPListing.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react'
```

Comment:
Need tests for this.

Reply:
Yup. I just went ahead and got a PR up, but this is very much a work in progress